ecs-init/config/common.go

```diff
@@ -202,23 +217,57 @@ func HostPKIDirPath() string {
 // AgentDockerLogDriverConfiguration returns a LogConfig object
 // suitable for used with the managed container.
 func AgentDockerLogDriverConfiguration() godocker.LogConfig {
-	maxSize := dockerJSONLogMaxSize
-	if fromEnv := os.Getenv(dockerJSONLogMaxSizeEnvVar); fromEnv != "" {
-		maxSize = fromEnv
+	exists := struct{}{}
+	validDrivers := map[string]struct{}{
+		"awslogs":    exists,
+		"fluentd":    exists,
+		"gelf":       exists,
+		"json-file":  exists,
+		"journald":   exists,
+		"logentries": exists,
+		"syslog":     exists,
+		"splunk":     exists,
```

Comment:
do you think it would be better if we initialize this variable outside of this function?